File: src/main/java/com/marknkamau/Exceptions/InputFileMissingExtension.java
package com.marknkamau.Exceptions;

public class InputFileMissingExtension extends BaseException {
    private static final String TITLE = "File missing extention";
    private static final String MISSING_INPUT_EXTENSION = "Input file does not have an extension.";

    public InputFileMissingExtension() {
        super(TITLE, MISSING_INPUT_EXTENSION);
    }
}

File: src/main/java/com/marknkamau/Exceptions/InputFileMissingExtension.kt
package com.marknkamau.Exceptions

class InputFileMissingExtension : BaseException(InputFileMissingExtension.TITLE, InputFileMissingExtension.MISSING_INPUT_EXTENSION) {
    companion object {
        private val TITLE = "File missing extension"
        private val MISSING_INPUT_EXTENSION = "Input file does not have an extension."
    }
}
